Bottom: 8ffee092bb21c0c147e575928b4c92b1d5cb176f
Top:    42b59cb7f554c623f97a8895f5dbbfcce916de9f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 15:56:24 +0100

Refresh of hid-gtk-rework-view-flip-code

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 0fad5d2..c8bc46c 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1370,7 +1370,6 @@ SwapSides (int argc, char **argv, Coord x, Coord y)
   int active_group = GetLayerGroupNumberByNumber (LayerStack[0]);
   int comp_group = GetLayerGroupNumberByNumber (component_silk_layer);
   int solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
-  int active_group = GetLayerGroupNumberByNumber (LayerStack[0]);
   bool comp_on = LAYER_PTR (PCB->LayerGroups.Entries[comp_group][0])->On;
   bool solder_on = LAYER_PTR (PCB->LayerGroups.Entries[solder_group][0])->On;
